lib/bin.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { resolve } from 'path';
+import { parseArgs } from '@pkgjs/parseargs';
```

Comment:
This was marked as stable in https://github.com/nodejs/node/pull/46718. Since it’s available as of Node 16, which is the oldest version supported by this project, I think it’s ok to import this from `node:util` now.